lib/chewy/search/request.rb

```diff
@@ -804,7 +804,7 @@ def exists?
       #   @return [Array<Chewy::Type>] result documents
       def find(*ids)
         ids = ids.flatten(1).map(&:to_s)
```

Comment:
Could you use `ids.flat_map(&:to_s)` here? https://ruby-doc.org/core-2.4.1/Enumerable.html#method-i-flat_map